Adyen/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,108 @@
+[!! Manage Adyen connection](../Integration/02_ManageAdyenConnection.md)
+
+
+# CONNECTIONS (Adyen)
+
+The CONNECTIONS tab provides an overview of all connections created for *Payments*. It also allows you to configure these connections.   
+For detailed information on general UI functions of the *CONNECTIONS* tab, see [Connections](../../Payments/UserInterface/08a_Connections.md) in the *Payments* documentation.
+
+
+## Edit Adyen connection &ndash; Credentials
+
+*Payments > Settings > Tab CONNECTIONS > Select Adyen&trade; connection > Tab Credentials*
+
+![Edit Adyen credentials](../../Assets/Screenshots/Adyen/Integration/EditCredentials.png "[Edit Adyen credentials]")
+
+- *Driver*  
+    Selected driver name. This drop-down list is read-only.
+
+**Credentials**
+
+The *Credentials* tab displays the connection credentials that you defined in Adyen&trade;. They are a copy of these credentials. 
+
+- *Checkout API endpoint*   
+   Endpoint of the checkout API that is managed in Adyen&trade;. For detailed information on the checkout API, see [Adyen Checkout API](https://docs.adyen.com/api-explorer/Checkout/71/overview "[https://docs.adyen.com/api-explorer/Checkout/71/overview]") in the Adyen&trade; documentation. For the testing environment it is `https://checkout-test.adyen.com/checkout/`, for the live account you get the endpoint directly from Adyen&trade;. 
```

Comment:
checkout API, Checkout API oder *Checkout API*. Ist es ein Eigenname? Soll es kursiv (UI-Element) sein oder nicht? Es ist nicht immer konsistent, vgl. Payment API endpoint unten und in Integration-Dateien.